plugins/VRPN/VRPNPlugin.h

```diff
@@ -46,7 +48,7 @@ class VRPNPlugin : public ExtensionPlugin
 
 private:
     PluginAPI* _api = nullptr;
-    Camera& _camera;
+    Camera* _camera;
```

Comment:
use `_api->getCamera()` instead, rather than having this pointer?

Reply:
Done